docs/admin/federation/index.md

```diff
@@ -137,11 +137,11 @@ cluster.
 Assuming our YAML file is located at `/cluster1/cluster.yaml`, we can run the following command to register this cluster:
 
 ```shell
-$ kubectl create -f /cluster1/cluster.yaml --cluster=federation-cluster
+$ kubectl create -f /cluster1/cluster.yaml --context=federation-cluster
```

Comment:
We want it today, because that's the name (`federation-cluster`) our scripts give to the context today - https://github.com/kubernetes/kubernetes/blob/master/federation/cluster/common.sh#L153.

I have been thinking about this lately and I think this should be configurable, may be with a default. It shouldn't be hard-coded at all. One could possibly have multiple federation API servers (contexts) running.